src/VerifierAlgorithm.ts

```diff
@@ -78,7 +78,7 @@ export function verifyES256(data: string, signature: string, authenticators: Ver
 
   const signer: VerificationMethod | undefined = fullPublicKeys.find((pk: VerificationMethod) => {
     try {
-      const pubBytes = extractPublicKeyBytes(pk)
+      const pubBytes = extractPublicKeyBytes(pk) as Uint8Array
```

Comment:
I wonder why this explicit type was needed here. Can you clarify?

Reply:
This was not needed. I have removed it.